salt/iaticdfdbackend.sls

```diff
@@ -0,0 +1,260 @@
+## Ubuntu 20 LTS only!
+#
+# Hetzner Oddness: After first install, postgres does not create it's own cluster!
+# Manually run: pg_createcluster 12 main
+# Then restart to make sure it is picked up correctly
+#
+
+{% from 'lib.sls' import createuser, apache, uwsgi, removeapache, removeuwsgi %}
+
+include:
+  - core
+  - apache
+  - uwsgi
+  - letsencrypt
+
+##################################################################### Normal User
+
+{% set user = 'iaticdfdbackend' %}
+{{ createuser(user) }}
+
+##################################################################### Logging User with special access
```

Comment:
Do we need special user with logging? We don't normally. We had that for IATI data store because of how logs were processed by another task later.